charts/pravega/README.md

```diff
@@ -42,13 +42,7 @@ The following table lists the configurable parameters of the Pravega chart and t
 | `zookeeperUri` | Zookeeper service address | `zk-client:2181` |
 | `externalAccess.enabled` | Enable Pravega external access | `false` |
 | `externalAccess.type` | Pravega external access type | `LoadBalancer` |
-| `bookkeeper.image.repository` | Image repo for Bookkeeper image | `pravega/bookkeeper` |
-| `bookkeeper.replicas` | Replicas for Bookkeeper | `3` |
-| `bookkeeper.storage.className` | Storage class for Bookkeeper volumes | `standard` |
-| `bookkeeper.storage.ledgerVolumeRequest` | Request storage for ledgerVolume | `10Gi` |
-| `bookkeeper.storage.journalVolumeRequest` | Request storage for journalVolume | `10Gi` |
-| `bookkeeper.storage.indexVolumeRequest` | Request storage for indexVolume | `10Gi` |
-| `bookkeeper.autoRecovery`| Enable Bookkeeper autoRecovery | `true` |
+| `bookkeeperUri` | URL for Bookkeeper Servers | `bookkeeper-client:3181` |
```

Comment:
I think that we cannot merge this PR until the Bookkeeper operator is developed, tested and released (initial release). Is that reasoning correct? Otherwise, users won't be able to deploy Pravega in the meantime.

Reply:
Actually, in the meantime I think it makes sense to have this feature: https://github.com/pravega/pravega-operator/tree/pravega-without-bk

Reply:
Making bookkeeper "optional" complicates a lot of things with p-operator which is why this option of removing it completely from p-operator was choosen. I would not recommend it at all. Once the BK PR is merged, I don't see why we should be blocked on merging this PR.